src/test/groovy/pl/cinema/server/reservation/WeekendPriceEvaluatorSpec.groovy

```diff
@@ -0,0 +1,30 @@
+package pl.cinema.server.reservation
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import spock.lang.Specification
+
+import java.time.LocalDateTime
+
+@SpringBootTest
```

Comment:
czy na prawdę potrzebujesz to testować integracyjnie i podnosić cały kontekst?

Reply:
miałem inną koncepcję wcześniej, ale patrząc na ten kod, to zupełnie nie, można przez new Stworzyć obiekt do testu i wywalić @SpringBootTest